flight/Modules/Battery/battery.c

```diff
@@ -133,6 +140,7 @@ static void batteryTask(void * parameters)
 		if (voltageADCPin >= 0) {
 			float adc_voltage = (float)PIOS_ADC_GetChannelVolt(voltageADCPin);
 			float scaled_voltage = 0.0f;
+			unsigned cells = batterySettings.NbCells;
```

Comment:
Could instead take away the " = batterySettings.NbCells" and then it can remain on the stack.